src/model.py

```diff
@@ -95,7 +111,27 @@ def _build_bad_words_list_for(_model_name: str) -> t.List[str]:
     # seems to have it quirks at the moment, but this is a rushed implementation
     # so I'm not handling that, hence the dumb return here.
     return ["Persona:", "Scenario:", "<START>"]
-
+    
+def _download_hf_files(repo_name: str) -> str:
+    '''
+    Downloads model files manually in the case that accelerate's load_checkpoint_and_dispatch() needs to be called.
+    :param repo_name: The name of the HuggingFace respository to download from.
+    
+    :return: The filepath to the index.json file for loading weights
+    '''
+    # NOTE (TG): Right now this is basically hardcoded for Pygmalion-6B.
```

Comment:
Minor, nitpicky suggestions just to keep style in sync with the rest of the code:

```suggestion
    '''
    Downloads model files manually in the case that accelerate's load_checkpoint_and_dispatch() needs to be called.

    :param repo_name: The name of the HuggingFace respository to download from.
    :return: The filepath to the index.json file for loading weights
    '''
    # NOTE(TG): Right now this is basically hardcoded for Pygmalion-6B.
```

Reply:
Looks good to me